<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapCanvas</class>
 <widget class="QWidget" name="MapCanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_30">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>20</width>
     <height>194</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color: transparent;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="smallerPushButtonCanvas">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/menor.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="betweenPushButtonCanvas">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/espacamento-entre-linhas.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="biggerPushButtonCanvas">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/maior-que.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addPushButtonCanvasIn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/sinal-de-adicao.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OkPushButtonCanvasIn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/verifica.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}
QPushButton:disabled{
	border: 2px solid rgba(100, 100, 100, 40);
	color: rgb(100, 100, 100, 40);
}
QPushButton:disabled{
	border: 2px solid rgb(100, 100, 100);
	background-color: rgb(100, 100, 100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetPushButtonCanvasIn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/reset.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}
QPushButton:disabled{
	border: 2px solid rgba(100, 100, 100, 40);
	color: rgb(100, 100, 100, 40);
}
QPushButton:disabled{
	border: 2px solid rgb(100, 100, 100);
	background-color: rgb(100, 100, 100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RevertPushButtonCanvasIn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/voltar.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}
QPushButton:disabled{
	border: 2px solid rgb(100, 100, 100);
	background-color: rgb(100, 100, 100);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MaximizePushButtonCanvasIn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/maximizar.png);
	padding-left: 2px;
	padding-right: 2px;
	padding-top: 2px; 
	padding-bottom: 2px;
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
